tests/TestHarness/transactions.py

```diff
@@ -37,9 +39,10 @@ def createAccount(self, account, creatorAccount, stakedDeposit=1000, waitForTran
         """Create account and return creation transactions. Return transaction json object.
         waitForTransBlock: wait on creation transaction id to appear in a block."""
         signStr = NodeosQueries.sign_str(sign, [ creatorAccount.activePublicKey ])
+        retryStr = f"--retry-num-blocks {self.retry_num_blocks_default}" if waitForTransBlock else ""
```

Comment:
Why not add `retry_num_blocks` to the `createAccount` argument list that defaults to the `retry_num_blocks_default` if not configured directly for the method call?  That way specific tests could control directly how much to retry?

Reply:
Good advice, and it should definitely be done that way when it's possible to do so.

Reply:
Added.